18-05-2020 03:46:18 TRACE CalculatorProject:? - read commands from System.in
18-05-2020 03:46:18 TRACE CommandFactory:? - file with classes was open successful
18-05-2020 03:47:28 TRACE Calculator:? - command name : DEFINE
18-05-2020 03:47:42 TRACE Calculator:? - command name : PUSH
18-05-2020 03:47:48 TRACE Calculator:? - command name : SQRT
18-05-2020 03:47:52 TRACE Calculator:? - command name : PRINT
18-05-2020 03:47:57 TRACE Calculator:? - command name : EXIT
18-05-2020 04:05:34 TRACE CalculatorProject:? - read commands from System.in
18-05-2020 04:05:34 TRACE CommandFactory:? - file with classes was open successful
18-05-2020 04:05:46 TRACE Calculator:? - command name : PUSH
18-05-2020 04:05:46 ERROR Calculator:36 - can't do operation : 
Exceptions.ArgumentsCountException: wrong count of arguments in push command
	at Commands.PushCommand.doCommand(PushCommand.java:14)
	at Calculator.doOperation(Calculator.java:67)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
18-05-2020 04:05:59 TRACE Calculator:? - command name : PUSH
18-05-2020 04:05:59 ERROR Calculator:36 - can't do operation : 
Exceptions.ArgumentTypeException: wrong type of argument in push command
	at Commands.PushCommand.doCommand(PushCommand.java:30)
	at Calculator.doOperation(Calculator.java:67)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
18-05-2020 04:06:08 TRACE Calculator:? - command name : PUSH
18-05-2020 04:06:12 TRACE Calculator:? - command name : SQRT
18-05-2020 04:06:19 TRACE Calculator:? - command name : PRINT
18-05-2020 04:06:22 TRACE Calculator:? - command name : POP
18-05-2020 04:06:24 TRACE Calculator:? - command name : PRINT
18-05-2020 04:06:24 ERROR Calculator:41 - can't do operation : 
java.util.EmptyStackException
	at java.util.Stack.peek(Stack.java:102)
	at Commands.PrintCommand.doCommand(PrintCommand.java:16)
	at Calculator.doOperation(Calculator.java:67)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
18-05-2020 04:06:30 TRACE Calculator:? - command name : EXIT
18-05-2020 04:11:55 TRACE CalculatorProject:32 - read commands from System.in
18-05-2020 04:11:55 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 04:12:27 TRACE Calculator:59 - command name : DEFINE
18-05-2020 04:12:31 TRACE Calculator:59 - command name : PUSH
18-05-2020 04:12:31 ERROR Calculator:36 - can't do operation : 
Exceptions.ArgumentsCountException: wrong count of arguments in push command
	at Commands.PushCommand.doCommand(PushCommand.java:14)
	at Calculator.doOperation(Calculator.java:67)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
18-05-2020 04:12:37 TRACE Calculator:59 - command name : PUSH
18-05-2020 04:12:37 ERROR Calculator:36 - can't do operation : 
Exceptions.ArgumentTypeException: wrong type of argument in push command
	at Commands.PushCommand.doCommand(PushCommand.java:30)
	at Calculator.doOperation(Calculator.java:67)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
18-05-2020 04:12:41 TRACE Calculator:59 - command name : PUSH
18-05-2020 04:12:45 TRACE Calculator:59 - command name : SQRT
18-05-2020 04:12:46 TRACE Calculator:59 - command name : PRINT
18-05-2020 04:12:51 TRACE Calculator:59 - command name : EXIT
18-05-2020 07:12:50 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 07:25:41 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 07:41:19 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 07:41:19 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 07:41:19 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 07:41:19 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 07:41:19 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 07:41:19 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 07:41:19 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 07:41:19 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 07:41:19 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:42:49 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:42:49 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:42:49 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:42:49 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:42:49 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:42:49 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:42:49 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:42:49 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:42:49 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:53:07 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:53:07 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:53:07 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:53:07 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:53:07 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:53:07 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:53:07 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:53:07 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:53:07 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:55:53 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:55:53 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:55:53 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:55:53 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:55:53 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:55:53 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:55:53 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:55:53 TRACE CommandFactory:35 - file with classes was open successful
18-05-2020 08:55:53 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:22:43 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:22:43 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at CalculatorTest.doOperation(CalculatorTest.java:24)
	at PopCommandTest.doOperation(PopCommandTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
01-06-2020 11:25:20 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:25:20 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at CalculatorTest.doOperation(CalculatorTest.java:24)
	at DefineCommandTest.doOperation(DefineCommandTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
01-06-2020 11:25:20 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:25:20 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at CalculatorTest.doOperation(CalculatorTest.java:24)
	at DivideByZeroTest.doOperation(DivideByZeroTest.java:17)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
01-06-2020 11:25:20 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:25:20 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at CalculatorTest.doOperation(CalculatorTest.java:24)
	at DivideCommandTest.doOperation(DivideCommandTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
01-06-2020 11:25:20 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:25:20 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at CalculatorTest.doOperation(CalculatorTest.java:24)
	at MultiplyCommandTest.doOperation(MultiplyCommandTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
01-06-2020 11:25:20 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:25:20 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at CalculatorTest.doOperation(CalculatorTest.java:24)
	at PopCommandTest.doOperation(PopCommandTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
01-06-2020 11:25:20 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:25:20 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at PushCommandTest.doOperation(PushCommandTest.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
01-06-2020 11:25:20 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:25:20 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at CalculatorTest.doOperation(CalculatorTest.java:24)
	at SqrtCommandTest.doOperation(SqrtCommandTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
01-06-2020 11:25:20 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:25:20 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at CalculatorTest.doOperation(CalculatorTest.java:24)
	at SubtractionCommandTest.doOperation(SubtractionCommandTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
01-06-2020 11:25:20 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:25:20 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at CalculatorTest.doOperation(CalculatorTest.java:24)
	at SumCommandTest.doOperation(SumCommandTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
01-06-2020 11:27:53 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:27:53 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:27:53 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:27:53 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:27:53 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:27:53 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:27:53 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:27:53 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:27:53 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:34:45 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:35:37 TRACE CalculatorProject:32 - read commands from System.in
01-06-2020 11:35:37 TRACE CommandFactory:35 - file with classes was open successful
01-06-2020 11:36:03 TRACE Calculator:59 - command name : define
01-06-2020 11:36:08 TRACE Calculator:59 - command name : push
01-06-2020 11:36:08 ERROR Calculator:36 - can't do operation : 
Exceptions.ArgumentsCountException: wrong count of arguments in push command
	at Commands.PushCommand.doCommand(PushCommand.java:14)
	at Calculator.doOperation(Calculator.java:67)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:36:17 TRACE Calculator:59 - command name : push
01-06-2020 11:36:22 TRACE Calculator:59 - command name : pop
01-06-2020 11:36:22 ERROR Calculator:36 - can't do operation : 
Exceptions.ArgumentsCountException: wrong count of arguments in pop command
	at Commands.PopCommand.doCommand(PopCommand.java:13)
	at Calculator.doOperation(Calculator.java:67)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:36:27 TRACE Calculator:59 - command name : pop
01-06-2020 11:36:33 TRACE Calculator:59 - command name : print
01-06-2020 11:36:33 ERROR Calculator:41 - can't do operation : 
java.util.EmptyStackException
	at java.util.Stack.peek(Stack.java:102)
	at Commands.PrintCommand.doCommand(PrintCommand.java:16)
	at Calculator.doOperation(Calculator.java:67)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:36:59 TRACE Calculator:59 - command name : push
01-06-2020 11:37:12 TRACE Calculator:59 - command name : push
01-06-2020 11:37:13 TRACE Calculator:59 - command name : +
01-06-2020 11:37:16 TRACE Calculator:59 - command name : print
01-06-2020 11:37:27 TRACE Calculator:59 - command name : pop
01-06-2020 11:37:29 TRACE Calculator:59 - command name : print
01-06-2020 11:37:29 ERROR Calculator:41 - can't do operation : 
java.util.EmptyStackException
	at java.util.Stack.peek(Stack.java:102)
	at Commands.PrintCommand.doCommand(PrintCommand.java:16)
	at Calculator.doOperation(Calculator.java:67)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:37:38 TRACE Calculator:59 - command name : exit
01-06-2020 11:37:38 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:37:38 ERROR Calculator:36 - can't do operation : 
Exceptions.UnknownCommandException: Unknown command
	at CommandFactory.buildCommand(CommandFactory.java:58)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:37:41 TRACE Calculator:59 - command name : exit
01-06-2020 11:37:41 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:37:41 ERROR Calculator:36 - can't do operation : 
Exceptions.UnknownCommandException: Unknown command
	at CommandFactory.buildCommand(CommandFactory.java:58)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:38:08 TRACE Calculator:59 - command name : exit
01-06-2020 11:38:08 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:38:08 ERROR Calculator:36 - can't do operation : 
Exceptions.UnknownCommandException: Unknown command
	at CommandFactory.buildCommand(CommandFactory.java:58)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:38:30 TRACE Calculator:59 - command name : exit
01-06-2020 11:38:30 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:38:30 ERROR Calculator:36 - can't do operation : 
Exceptions.UnknownCommandException: Unknown command
	at CommandFactory.buildCommand(CommandFactory.java:58)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:38:31 TRACE Calculator:59 - command name : 
01-06-2020 11:38:31 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:38:31 ERROR Calculator:36 - can't do operation : 
Exceptions.UnknownCommandException: Unknown command
	at CommandFactory.buildCommand(CommandFactory.java:58)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:38:33 TRACE Calculator:59 - command name : exit
01-06-2020 11:38:33 ERROR CommandFactory:57 - try to do unknown command : 
java.lang.NullPointerException
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at CommandFactory.buildCommand(CommandFactory.java:51)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
01-06-2020 11:38:33 ERROR Calculator:36 - can't do operation : 
Exceptions.UnknownCommandException: Unknown command
	at CommandFactory.buildCommand(CommandFactory.java:58)
	at Calculator.doOperation(Calculator.java:66)
	at Calculator.startCalculate(Calculator.java:32)
	at CalculatorProject.main(CalculatorProject.java:36)
